### ISSUE TITLE
mushroom-shaped mass on fundus

### ISSUE THREAD

**Issue by LOURGHI** [obophenotype/human-phenotype-ontology#3720]
_**Please consider a new term for mushroom-shaped mass on fundus observed in Uveal melanoma.
PMID:27800275.
Superclass:Abnormality of the fundus??**_

**Comment by pnrobinson** [comment]
Was the mushroom shaped mass simply the melanoma? In this case, we should create a term for uveal melanoma.

**Comment by pnrobinson** [comment]
I am seeing we have a term Intraocular melanoma (that has as a synonym Uveal melanoma)
https://hpo.jax.org/app/browse/term/HP:0007716

I think that Uveal melanoma should be the primary label, I will change it. I am pretty sure that the mass was simply the melanoma -- but please reopen if necessary.
